debian/control

```diff
@@ -29,3 +29,14 @@ Description: high performance general RPC framework
  transparently, and makes it easier to build connected systems.
  .
  This package provides the gRPC C bindings.
+
+Package: libgrpc0-cil
```

Comment:
No, spelling is correct and consistent with other such packages. Check out
~~~
apt-cache search lib\.\*-cil
~~~